ledger/ledger-on-sql/src/app/scala/com/daml/ledger/on/sql/SqlLedgerFactory.scala

```diff
@@ -28,22 +29,24 @@ object SqlLedgerFactory extends LedgerFactory[SqlLedgerReaderWriter, ExtraConfig
   override def manipulateConfig(config: Config[ExtraConfig]): Config[ExtraConfig] =
     config.copy(participants = config.participants.map(_.copy(allowExistingSchemaForIndex = true)))
 
-  override def owner(
+  override def readWriterServiceOwner(
       config: Config[ExtraConfig],
       participantConfig: ParticipantConfig
   )(
       implicit executionContext: ExecutionContext,
       materializer: Materializer,
       logCtx: LoggingContext,
-  ): ResourceOwner[SqlLedgerReaderWriter] = {
+  ): ResourceOwner[KeyValueParticipantState] = {
```

Comment:
Why `KeyValueParticipantState` and not `ReadWrite(r)Service`? `ResourceOwner` is covariant.